### PR TITLE
IOS-XE improvement for ISIS model

### PR DESCRIPTION
**Pull request by Tristou27**
## Description

When ISIS model was called, itwas not working correctly for IOS-XE in some cases. This PR aims at fixing those cases.

## Motivation and Context

Following Issue [36](https://github.com/CiscoTestAutomation/genielibs/issues/36)  opened on genielibs (my mistake) : SchemaMissingKeyError while trying to learn ISIS information.

After investigation on ISIS parsers : 
- ShowClnsTraffic parser was not able to handle ISIS Level2-only nodes, as "level-1" infos were mandatory in the schema. This is fixed by setting both level1 and level2 as optional.
- ShowIsisDatabaseDetail was not able to retrieve information, ending on a schema validation error state. This was due to a possible field not handled correctly and introducing bad structure in the output dictionary. This is fixed with the correct handling of the "IPV4 Interarea Reachability" information.

### Additional changes:
When trying to execute the code and update the json, errors appeared in Viptela/ShowCloudexpressApplication. This was due to duplicated lines after the end of a method. Duplicated lines have been removed. 

## Impact (If any)
No impact seen as the changes are just added, no deletion

## Screenshots:

```
(genie_dev) {20-11-17 15:50}nietb4:~/fork/genieparser/tests@master ttr% python -m unittest -v
(...)
----------------------------------------------------------------------
Ran 1825 tests in 33.738s

OK
```
## Checklist:

- [x] I have updated the changelog.
- [ ] I have updated the documentation (If applicable).
- [x] I have added tests to cover my changes (If applicable).
- [x] All new and existing tests passed.
- [x] All new code passed compilation.
- [x] I have updated the global Json schema

### CODE REVIEW

File: changelog/changelog_updateparser_clnsisis_20201117114236.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--------------------------------------------------------------------------------
```
Comment (xiaoxinz-cisco) [inline changelog/changelog_updateparser_clnsisis_20201117114236.rst:1]: this block can be removed

File: src/genie/libs/parser/iosxe/tests/ShowClnsTraffic/cli/equal/golden_output_2_expected.py [hunk 2]
```diff
@@ -59,9 +59,9 @@
                 "lsp_retransmissions": 0,
                 "csnp": {"level-1": {"sent": 308588, "rcvd": 149877}},
                 "psnp": {"level-1": {"sent": 0, "rcvd": 0}},
-                "dr_election": {"level-1": 9},
-                "spf_calculation": {"level-1": 1565},
-                "partial_route_calculation": {"level-1": 4},
+                "dr_election": {"level-1": 9, "level-2":7},
+                "spf_calculation": {"level-1": 1565, "level-2": 1324},
+                "partial_route_calculation": {"level-1": 4, "level-1": 6},
```
Comment (xiaoxinz-cisco) [inline src/genie/libs/parser/iosxe/tests/ShowClnsTraffic/cli/equal/golden_output_2_expected.py:64]: `'level-2':6`*?

**Review by xiaoxinz-cisco — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/tests/ShowClnsTraffic/cli/equal/golden_output_2_expected.py (hunk 2 above)
Comment (Tristou27) [reply src/genie/libs/parser/iosxe/tests/ShowClnsTraffic/cli/equal/golden_output_2_expected.py]: Sure, typo. Thanks. It has been corrected

File: changelog/changelog_updateparser_clnsisis_20201117114236.rst (hunk 1 above)
Comment (Tristou27) [reply changelog/changelog_updateparser_clnsisis_20201117114236.rst:1]: Of course, thanks.

**Review by xiaoxinz-cisco — APPROVED** [review]
Thanks, it looks good to go. We're looking for your next contribution ;D

**Review by danielgraziano — APPROVED** [review]

**Pull request merged**